Sources/SQLKit/Query/SQLAlterTable.swift

```diff
@@ -23,22 +23,41 @@ public struct SQLAlterTable: SQLExpression {
     }
     
     public func serialize(to serializer: inout SQLSerializer) {
+        let syntax = serializer.dialect.alterTableSyntax
+
+        if !syntax.allowsBatch && self.addColumns.count + self.modifyColumns.count + self.dropColumns.count > 1 {
+            serializer.database.logger.warning("Database does not support batch table alterations. You will need to rewrite as individual alter statements.")
+        }
+
+        if syntax.alterColumnDefinitionClause == nil && self.modifyColumns.count > 0 {
+            serializer.database.logger.warning("Database does not support column modifications. You will need to rewrite as drop and add clauses.")
```

Comment:
Based on prior conversation, this should maybe actually be a `fatalError`, right? In this case, the alternative is omitting any `modifyColumns` which results in a different database structure than intended without any SQL errors being produced.

Reply:
Hmm good point. We should probably either:
- `fatalError`
- Serialize the modifications using a default `SQLRaw("MODIFY")` keyword.

I lean toward the latter since that's similar to how the batch is working. Warning + invalid SQL. I'm not super passionate either way though.

Reply:
ok, went your preferred route, having an equal amount of ambivalence myself.